_posts/2020-10-10-temperature-water-content-and-polishing-effects-on-koji.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Temperature, water content, and polishing effects on koji
+author: keith
+categories:
+- koji
+tags:
+- koji
+- amylase
+- enzymes
+- rice
+- featured
+- water-content
+---
+
+I always wondered if researchers had played around with koji growing parameters - turns out they have. This post will be the first of a few going over these studies - starting with temperature, water content, and polishing rate.
+
+### Overview
+
+Koji-making is all about optimizing growth conditions for the specific application you're after. Although koji-makers had centuries of knowledge about optimal growth conditions, it wasn't examined  scientifically until the 20th century.
+
+Everything in this post comes from the Brewing Society of Japan, and its academic journal (which has been running from 1906 to today)  - more on that at the end of the post. The experiments below were run mostly in the the 1970s and the 1980s by researchers looking to explain and quantify different aspects of koji-making.
+
+### Temperature
+
+One of the most obvious parameters that researchers tested was the **cultivation temperature**. The growth of A. oryzae on white rice was tested at temperatures between 30°C and 40°C - and this is what they found:
+
+![Table 1](/assets/images/temp_effect/Table 1.png)
+*Effect of culture temperature on enzyme activity ([1] - 10mL O<sub>2</sub> consumed/ g koji and [2] at maximum cell mass)*
+
+Of course, the researchers know that growing koji for a fixed time at different temperatures would not be a fair comparison. At different temperatures, the koji's growth rate (μ, measured in h<sup>-1</sup>) is also different. Thus, enzyme activity was compared at two points different points:
+
+1. When the koji reaches 10mL oxygen/g dry weight of koji, and;
```

Comment:
yea this line - I don't know what "10mL oxygen/g dry weight of koji" represents

Reply:
Agreed I worded it confusingly.

10mL oxygen consumed per gram of koji (dry weight basis) 
Let me think of a better way to word this ; maybe 
"When the koji has consumed 10mL of oxygen per gram of koji (dry weight basis)"